services/calculator/src/__tests__/express.spec.ts

```diff
@@ -144,7 +170,7 @@ describe("express app", () => {
     });
 
     it("should throw 404 error", async () => {
-      await expect(t.get("/calculator")).rejects.toThrowError(
+      await expect(t.get("/calculato")).rejects.toThrowError(
```

Comment:
I'm guessing this is /calculator

Reply:
that's to throw a 404...just to fix a failing test after adding the snapshot endpoint

Reply:
understood! great!